bin/p2-exec/main.go

```diff
@@ -80,6 +92,27 @@ func main() {
 		}
 	}
 
+	if *podID != "" {
+		if *launchableName != "" {
+			log.Fatalf("Specified pod name %q, but no launchable name was specified", *podID)
+		}
+		if *cgroupName != "" {
+			log.Fatalf("Specified pod name %q, but no cgroupname name was specified", *podID)
+		}
+
+		if config := os.Getenv("CONFIG_PATH"); config != "" {
```

Comment:
What is the value of this environment variable here at this point? I suspect this is the preparer's CONFIG_PATH, not the pod we are deploying.

Further to this point, is this approach sound in cases where we've not deployed this pod before? How about when we change the values of the cgroup config? I think these cases are handled already, but I'd like to be certain.

Reply:
it's going to have `CONFIG_PATH` of the pod being deployed because of the `os.Setenv` loop on line 82, but maybe that relationship should be made more explicit. Probably worth crashing or setting some kind of cgroup default if there's no `CONFIG_PATH` in the env dir?